jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l

```diff
@@ -561,6 +560,7 @@
             (:auto-dock-failure
               '(lambda (userdata)
                  (report-auto-dock-failure)
+                 (ros::service-call "shutdown" (instance std_srvs::EmptyRequest :init))
```

Comment:
I think `:room-light-off` does not run when auto-dock fails, if shutdown is starting in this position.